src/www/status_dhcpv6_leases.php

```diff
@@ -454,7 +476,7 @@ function parse_duid($duid_string)
                     foreach ($config['dhcpdv6'] as $dhcpif => $dhcpifconf) {
                         if (isset($dhcpifconf['staticmap'])) {
                             foreach ($dhcpifconf['staticmap'] as $staticent) {
-                                if ($data['ip'] == $staticent['ipaddr']) {
+                                if ($data['ip'] == $staticent['ipaddrv6']) {
```

Comment:
looks like a fix... can we separate the $mac_man use and this into a separate PR?